docs/cloud-native-security/vuln-management-overview.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[vuln-management-overview]]
+= Cloud native vulnerability management
+
+Elastic's Cloud Native Vulnerability Management (CNVM) feature helps you identify known vulnerabilities in your cloud workloads.
+
+Setup uses infrastructure as code. For instructions, refer to <<vuln-management-get-started, Get started with Cloud Native Vulnerability Management>>.
+
+NOTE: CNVM currently only supports AWS EC2 Linux workloads.
+
+.Requirements
+[sidebar]
+--
+* CNVM is available to all {ecloud} users. On-premise deployments require an https://www.elastic.co/pricing[Enterprise subscription].
+* {stack} and {agent} versions 8.8 or higher.
+* To view vulnerability scan findings, you need at least `read` privileges for the following indices:
+** `logs-cloud_security_posture.vulnerabilities-*`
+** `logs-cloud_security_posture.vulnerabilities_latest-*`
+--
+
+[discrete]
+[[vuln-management-overview-how-it-works]]
+== How CNVM works
+
+During setup, you will use an infrastructure as code provisioning template to create a new virtual machine (VM) in the cloud region you wish to scan. This VM installs {agent} and the Cloud Native Vulnerability Management (CNVM) integration, and conducts all vulnerability scanning.
+
+The CNVM integration uses https://github.com/aquasecurity/trivy[Trivy], a comprehensive open-source security scanner, to scan cloud workloads and identify security vulnerabilities. During each scan, the VM running the integration takes a snapshot of all cloud workloads in its region using the snapshot APIs of the cloud service provider, and analyzes them for vulnerabilities using Trivy. Therefore, scanning does not use resources on the VMs being scanned. All resource usage occurs on the VM installed during CNVM setup.
+
+The scanning process begins immediately upon deployment, then repeats every twenty-four hours. After each scan, the integration sends the discovered vulnerabilities to {es}, where they appear in the *Vulnerabilities* tab of the <<vuln-management-findings, Findings page>>.
+
+NOTE: Environments with more VMs take longer to scan.
+
+// == Use Cases
```

Comment:
Remove? If the plan is to eventually expand the page with use cases, just add the heading at that time, not now.